test/e2e/sched/install/install_test.go

```diff
@@ -97,7 +97,7 @@ var _ = Describe("[Scheduler] install", func() {
 			Expect(err).NotTo(HaveOccurred())
 			Expect(podList).ToNot(BeEmpty(), "cannot find any pods for DP %s/%s", deployment.Namespace, deployment.Name)
 
-			nodeList, err := nodes.GetControlPlane(e2eclient.Client, configuration.Plat)
+			nodeList, err := nodes.GetControlPlane(e2eclient.Client, context.TODO(), configuration.Plat)
```

Comment:
Is `context.TODO()` ok here because it's only for testing?

Reply:
The idea is to first bubble up the context, so the top-level code can either use a proper context or move `context.TODO()` at top-level.
Here we do the latter because I still need to learn how to pass contexts fron ginkgo code. I'll fix with a future PR.